python/cudf/CMakeLists.txt

```diff
@@ -26,40 +38,73 @@ project(
             # various linking options for the python library is hardcoded to build with C, so until
             # that is fixed we need to keep C.
             C CXX
+            # TODO: This will not be necessary once we upgrade to CMake 3.22, which will pull in the required
```

Comment:
Similarly, this workaround can be removed because https://github.com/rapidsai/cudf/pull/11751 is merged.